lib/fluent/plugin/in_cloudwatch_logs.rb

```diff
@@ -135,16 +135,19 @@ def run
     end
 
     def emit(stream, event)
-      if @parser
-        @parser.parse(event.message) {|time, record|
+        if @parser
+          @parser.parse(event.message) {|time,record|
+            if @use_aws_timestamp
```

Comment:
If you intend to use use_aws_timestamp, you probably need to add it in the section with all the config_param by line 12 -27.  Your addition should be like:

```
 config_param :use_aws_timestamp, :bool, default: false
```

👍 for the change btw

Reply:
Good catch! Missed commit that I just added!